src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -641,7 +642,7 @@ public boolean deleteSnapshot(Snapshot snapshot) throws InterruptedException {
         } else {
             logger.warn("Snapshot is missing sources: " + snapshot.getName());
         }
-        return bigQueryProject.deleteDataset(snapshot.getName());
+        return bigQueryProject.deleteDataset(snapshot.getName());*/
```

Comment:
[TODO - remove these block comments] - I'm trying to prove that we need this chunk of code. Without this code, BQ doesn't automatically remove the ACLs from the source dataset.